releng/org.eclipse.xtext.sdk.target/org.eclipse.xtext.sdk.target.target

```diff
@@ -140,22 +140,22 @@
 		<unit id="com.google.gson.source" version="2.7.0.v20170129-0911"/>
 		<unit id="org.antlr.runtime" version="3.2.0.v201101311130"/>
 		<unit id="org.antlr.runtime.source" version="3.2.0.v201101311130"/>
-		<unit id="org.junit" version="4.12.0.v201504281640"/>
-		<unit id="org.junit.source" version="4.12.0.v201504281640"/>
-		<unit id="org.junit.jupiter.api" version="5.3.1.v20181005-1442"/>
-		<unit id="org.junit.jupiter.api.source" version="5.3.1.v20181005-1442"/>
-		<unit id="org.junit.jupiter.engine" version="5.3.1.v20181005-1442"/>
-		<unit id="org.junit.jupiter.engine.source" version="5.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.commons" version="1.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.commons.source" version="1.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.engine" version="1.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.engine.source" version="1.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.launcher" version="1.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.launcher.source" version="1.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.runner" version="1.3.1.v20181005-1442"/>
-		<unit id="org.junit.platform.runner.source" version="1.3.1.v20181005-1442"/>
```

Comment:
i am not sure if i like this. i think having explicit versions was conscious decision

Reply:
i dont know when and by which critiera orbit maintains single or multiple versions for things

Reply:
It certainly helps to reduce the build time.
Wouldn't the versions that we get from orbit be checked against the manifest versions anyways?

Reply:
yes and no. we dont have ranges. so will will pull latest gratest anyway